web-app-router/scenes/Portal/layout/CreateAppDialog/index.tsx

```diff
@@ -124,10 +124,10 @@ export const CreateAppDialog = (props: DialogProps) => {
 
   return (
     <Dialog open={props.open} onClose={onClose} className="z-50">
-      <DialogPanel className={clsx("fixed inset-0 p-4", props.className)}>
-        <div className="grid grid-rows-auto/1fr items-center">
-          <SizingWrapper>
-            <header className="w-full flex justify-between items-center min-h-9">
+      <DialogPanel className={clsx("fixed inset-0 p-0", props.className)}>
```

Comment:
can we move the overflow-y-auto to the panel? That way the scroll bar is flush with the side of the screen 
```suggestion
      <DialogPanel className={clsx("fixed inset-0 p-0 overflow-y-auto", props.className)}>
```